scripts/ml/train_risk_model.py

```
@@ -0,0 +1,382 @@
+    df["patient_status_encoded"] = df["patient_status"].apply(
+        lambda x: encode_categorical(x, CATEGORICAL_MAPS["patientStatus"])
+    )
+    df["has_asha_assigned"] = (df["has_asha_assigned"] == "true").astype(int)
+    df["is_transferred"] = (df["transferred"] == "true").astype(int)
+
+    # --- Proxy label: non-adherent = 1, adherent = 0 ---
+    df["label"] = (
+        (df["follow_up_count"] == 0) & (df["days_since_registration"] > threshold_days)
+    ).astype(int)
+
+    print(f"   Label distribution:")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)